src/components/common/Modal.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import styled, { ThemeProvider } from "styled-components";
+import PropTypes from "prop-types";
+
+import theme from "../../styles/theme";
+
+export default function Modal({ children }) {
+  const navigate = useNavigate();
+
+  const handleExitClick = event => {
+    if (event.target !== event.currentTarget) {
+      return;
+    }
+
+    navigate(-1);
+  };
+
+  useEffect(() => {
+    const $modalRoot = document.getElementById("modal-root");
+
+    $modalRoot.style.position = "relative";
+    document.body.style.overflow = "hidden";
+
+    return () => {
+      document.body.style.overflow = "unset";
+    };
```

Comment:
body도 이렇게 하는건 어떤가요??

```suggestion
    const $modalRoot = document.getElementById("modal-root");
    const $body = document.body;

    $modalRoot.style.position = "relative";
    $body.style.overflow = "hidden";
    
    return () => {
      $body.style.overflow = "unset";
    };
```

Reply:
좋은 생각인 것 같아요~! :)